G1ANT.Chrome.Driver/ChromeClient.cs

```diff
@@ -0,0 +1,67 @@
+﻿using G1ANT.Browser.Driver.Actions;
+using G1ANT.Browser.Driver.Data;
+using G1ANT.Browser.Driver.Services;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace G1ANT.Chrome.Driver
+{
+    public class ChromeClient : BrowserClient
+    {
+        protected override string ServerName => Configuration.ServerName;
+
+        public ChromeClient(BrowserService browserService) : base(browserService)
+        {
+        }
+
+        private bool urlLoaded = false;
+        public override void OpenBrowserWithUrl(string url, string extraArguments = "")
+        {
+            urlLoaded = false;
+            BrowserService.EventsService.OnTabUpdated += TabUpdatedHandler;
+
+            Process.Start("chrome.exe", $"\"{url}\" {extraArguments}");
```

Comment:
Brave, Edge, etc.

Reply:
Will be added during Edge/Brave implementation